### PR TITLE
Fix Gerrit adapater to always use throttling.

### PR DESCRIPTION
**Pull request by cjwagner**
/assign @timwangmusic @listx @airbornepony 

The `UpdateClients` function was removing our round tripper wrapper :sob:

### CODE REVIEW

File: prow/gerrit/client/client.go [hunk 1]
```diff
@@ -124,6 +124,8 @@ type Client struct {
 	// map of instance to gerrit account
 	accounts map[string]*gerrit.AccountInfo
 
+	http http.Client
```
Comment (timwangmusic) [inline prow/gerrit/client/client.go:127]: consider call this `httpClient`

Comment (cjwagner) [reply prow/gerrit/client/client.go]: Done.

**Review by timwangmusic — APPROVED** [review]
LGTM

File: prow/gerrit/client/client.go [hunk 2]
```diff
@@ -320,6 +315,23 @@ func (c *Client) Authenticate(cookiefilePath, tokenPath string) {
 	}
 }
 
+func (c *Client) newInstanceHandler(instance string, projects map[string]*config.GerritQueryFilter) (*gerritInstanceHandler, error) {
```
Comment (airbornepony) [inline prow/gerrit/client/client.go:318]: Wondering what "Handler" means here. Can it be instanceClient() or so?

**Review by airbornepony — APPROVED** [review]

File: prow/gerrit/client/client.go (hunk 2 above)
Comment (airbornepony) [reply prow/gerrit/client/client.go:318]: I see this is consistent with the gerritInstanceHandler struct name.

**Pull request merged**